packages/bpk-component-modal/src/BpkModalV2/README.md

```diff
@@ -0,0 +1,76 @@
+# bpk-component-modal
+
+> Backpack modal component version 2.
```

Comment:
Could you also add a note to indicate that this component is experimental? Just in case anyone comes across this so they would know it's not stable.
It's the first experimental component, so feel free to choose how you want to write it, we don't have an example for it 😄 You can use an emoji too like 💡 or 🧪 if you want!